qiskit_jku_provider/qasm_simulator_jku.py

```diff
@@ -23,13 +23,16 @@
 from collections import OrderedDict, Counter
 import numpy as np
 
-from qiskit.backends import BaseBackend
+#from qiskit.backends import BaseBackend
 from qiskit.backends.aer import AerJob
 from qiskit.backends.aer._simulatorerror import SimulatorError
```

Comment:
Also, can you replace this import with your own error class, inheriting from `QiskitError`, as in https://github.com/Qiskit/qiskit-projectq-provider/blob/master/qiskit_addon_projectq/projectqsimulatorerror.py ?

Reply:
Sure thing, done.